distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Worker.java

```diff
@@ -377,7 +377,7 @@ private void onWorkPooledJobReassigned() throws Exception {
                     if (!isWorkerShutdown) {
                         ScheduledFuture<?> scheduledFuture =
                                 jobReschedulableScheduler.schedule(
-                                        DynamicProperty.of(newMultiJob::getSchedule),
+                                        DynamicProperty.of(newMultiJob.getSchedule()),
```

Comment:
you are changing logic here: of - for constant, delegated for dynamic. We need dynamic way.